site/docs/latest/api/ledger-api.md

```diff
@@ -20,7 +20,7 @@ If you're using [Maven](https://maven.apache.org/), add this to your [`pom.xml`]
 
 ```xml
 <!-- in your <properties> block -->
-<bookkeeper.version>{{ site.latest_version }}</bookkeeper.version>
+<bookkeeper.version>4.6.0</bookkeeper.version>
```

Comment:
this should be `{{ site.latest_version}}`

Reply:
done